src/checks/quote_character.rs

```diff
@@ -24,7 +24,7 @@ impl Default for QuoteCharacterChecker<'_> {
 impl Check for QuoteCharacterChecker<'_> {
     fn run(&mut self, line: &LineEntry) -> Option<Warning> {
         let val = line.get_value()?;
-        if val.contains("\\n") || val.contains(char::is_whitespace) {
+        if val.contains("\\n") || val.contains(char::is_whitespace) || val.contains("$") {
```

Comment:
Add this case(with `$`) to tests, please 🙂

Reply:
added